charts/revere/values.yaml

```diff
@@ -0,0 +1,211 @@
+name: revere
+
+imageConfig:
+  # -- (string) Image repository
+  repository: us-central1-docker.pkg.dev/dsp-artifact-registry/revere/revere
+  # -- (string) Image tag
+  # @default -- global.applicationVersion
+  tag: null
+  # -- (string) When to pull images
+  imagePullPolicy: Always
+
+# -- (number) Number of replicas for the deployment
+replicas: 3
+
+# -- (map) Annotations for application pods
+annotations: {}
+
+# -- (string) Optional override for the pod's runtimeClass
+runtimeClassName:
+
+resources:
+  requests:
+    # -- (string) Number of CPU units to request for the deployment
+    cpu: 500m
+    # -- (string) Memory to request for the deployment
+    memory: 4Gi
+  limits:
+    # -- (string) Number of CPU units to limit the deployment to
+    cpu: 500m
+    # -- (string) Memory to limit the deployment to
+    memory: 4Gi
+
+probes:
+  readiness:
+    # -- (boolean) If the readiness probe should be enabled
+    enabled: false
+    # -- (object) Spec for the readiness probe
+    spec:
+  liveness:
+    # -- (boolean) If the liveness probe should be enabled
+    enabled: false
+    # -- (object) Spec for the liveness probe
+    spec:
+  startup:
+    # -- (boolean) If the startup probe should be enabled
+    enabled: false
+    # -- (object) Spec for the startup probe
+    spec:
+
+secrets:
+  gcpServiceAccount:
+    workloadIdentity:
+      # -- (boolean) If workloadIdentity should be used for the GCP SA
+      enabled: false
+      # -- (string) ID of the GCP SA to use
+      accountName:
+      # -- (string) ID (not the number) of the GCP project the SA is in
+      projectId:
+    secretsManager:
+      # -- (boolean) If secretsManager should be used for the GCP SA
+      enabled: false
+      # -- (string) Path within Vault where the SA's key is stored
+      vaultPath:
+      # -- (string) Field name within the secret for the SA's key
+      vaultKey:
+      # -- (string) Encoding of the Vault field (either `text` or `base64`)
+      sourceEncoding: base64
+  statuspageApiKey:
+    secretsManager:
+      # -- (boolean) If secretsManager should be used for the API key
+      enabled: false
+      # -- (string) Path within Vault where the API key is stored
+      vaultPath:
+      # -- (string) Field name within the secret for the API key
+      vaultKey:
+      # -- (string) Encoding of the Vault field (either `text` or `base64`)
+      sourceEncoding: text
+
+networkPolicy:
+  ingress:
+    # -- (boolean) If the NetworkPolicy should be enabled for ingress
+    enabled: true
+    # -- (list) CIDR ranges to permit for ingress
+    cidrAllowList:
+    # -- (list) Posts/protocols to permit for ingress
+    portAllowList:
+  egress:
+    # -- (boolean) If the NetworkPolicy should be enabled for egress
+    enabled: true
+    # -- (list) CIDR ranges to permit for egress
+    cidrAllowList:
+    # Atlassian for Statuspage, per https://ip-ranges.atlassian.com/
+    - 2600:1f18:2146:e300::/56
```

Comment:
Not sure how I feel about encoding all of this as default values. I think for the helm chart I'd prefer just an empty list as default. Then we populate this in terra-helmfile.

Reply:
That's a really good point--I think I did all the research to flesh it out and didn't want to lose the info here. I can split this out to a terra-helm-definitions PR on Monday

Reply:
Also we may want to take advantage of Chelsea's recent helmfile changes and deploy this via terra-helmfile rather than terra-helm-definitions.

Reply:
Good idea. I have a -definitions PR open [here](https://github.com/broadinstitute/terra-helm-definitions/pull/67) just as an example but I'm marking it as a draft--I think I might tackle that once the chart's design is a bit more fleshed out but before merging anything, just so we can sorta be reviewing one thing at a time.